doc/staging.md

```diff
@@ -145,7 +135,7 @@ such as `config/core/<your-username>.yaml`:
 
 ```yaml
 storage:
-  early-access-azure-<your-username>:
+  staging-azure-<your-username>:
     storage_type: azure
     base_url: https://kciapistagingstorage1.file.core.windows.net/
     sas_public_token: "?sv=2022-11-02&ss=f&srt=sco&sp=r&se=2024-10-17T19:19:12Z&st=2023-10-17T11:19:12Z&spr=https&sig=sLmFlvZHXRrZsSGubsDUIvTiv%2BtzgDq6vALfkrtWnv8%3D"
```

Comment:
@nuclearcat The same question as above for this.

Reply:
Yes, whole point probably is to rework concept of staging. I dont think we will give separate API accounts anymore (and especially storage).

Reply:
Okay, thanks for confirming.